src/auspex/instruments/X6.py

```diff
@@ -143,6 +152,12 @@ def disconnect(self):
     def set_all(self, settings_dict):
         # Call the non-channel commands
         super(X6, self).set_all(settings_dict)
+        # Set data for testing
+        if self.ideal_data:
+            try:
+                self.ideal_data = np.load(os.path.abspath(self.ideal_data+'.npy'))
+            except:
```

Comment:
If None above, we could ditch this None here and instead throw a proper error if the file doesn't exist.